bed/methods_test.go

```diff
@@ -66,3 +74,28 @@ func TestBed_UpdateCoord(t *testing.T) {
 		}
 	}
 }
+
+//	Tests for methods that use: type BedSlice []*Bed
+func TestBedSlice_Len(t *testing.T) {
+	var allBed BedSlice
+	for _, v := range BedMethodTests {
+		pointer := &v.b //must create a pointer because type BedSlice is []*Bed
+		allBed = append(allBed, pointer)
```

Comment:
No issue here, but just for reference you can append a pointer without going through an intermediate variable.
`allBed = append(allBed, &v.b)`